backend/src/db-utils.js

```diff
@@ -528,6 +528,24 @@ const queryPartnerUniqueUserAmount = async () => {
   );
 };
 
+// bridge query
+const queryBridgeUser = async (accountId) => {
+  return await queryRows(
+    [
+      `select 
+        distinct transactions.signer_account_id as holder
+      from receipts 
+      join action_receipt_actions on receipts.receipt_id = action_receipt_actions.receipt_id  
+      join transactions on transactions.transaction_hash = receipts.originated_from_transaction_hash 
+      where receipts.receiver_account_id = :account_id
+      and action_receipt_actions.action_kind = 'FUNCTION_CALL' 
+      and action_receipt_actions.args->> 'method_name' = 'mint'`,
```

Comment:
1. You probably only want successful receipts
2. nit: Please use upper-case for the SQL keywords (follow the code style)

Reply:
1. do not need that since query also needs failure one
2. fix the case pro